sql/total_nft.sql

```diff
@@ -0,0 +1,7 @@
+CREATE INDEX IF NOT EXISTS idx_nft_owner ON nft (owner);
+CREATE INDEX IF NOT EXISTS idx_iscn_iscn_id_prefix ON iscn (iscn_id_prefix);
+CREATE INDEX IF NOT EXISTS idx_nft_class_iscn_id_prefix ON nft_class (parent_iscn_id_prefix);
```

Comment:
Do we need schema upgrade?

Reply:
These are already included in v003.sql in #33

Reply:
Updated some index in v004.sql